core/src/main/java/nl/nn/adapterframework/soap/Wsdl.java

```diff
@@ -131,7 +140,7 @@
 
     private List<String> warnings = new ArrayList<String>();
 
-    public Wsdl(PipeLine pipeLine) {
+    public Wsdl(PipeLine pipeLine, IPipeLineSession session) {
```

Comment:
Nog één keer iets op te merken:
Je hebt hier het argument IPipeLineSession session geintroduceerd, alleen maar om de requestURL op te halen. 
Deze Wsdl heeft verder helemaal niets met deze sessie.
Ik stel voor dat je in plaats hiervan een argument 'String generationInfo' opneemt, zodat je op regel 321 zoiets kan doen als:
'documentation = "Generated" + (generationInfo!= null ? " " + generationInfo: "") + " as " + ... '
En ook een constructor zonder dat argument, zodat je niet altijd null hoeft mee te geven als je er niet in geinteresseerd bent.